src/components/__tests__/ChallengeInactiveInfo.cy.js

```diff
@@ -0,0 +1,106 @@
+import { colors } from 'quasar';
+import ChallengeInactiveInfo from 'components/homepage/ChallengeInactiveInfo.vue';
+import { i18n } from '../../boot/i18n';
+
+// colors
+const { getPaletteColor, changeAlpha } = colors;
+const white = getPaletteColor('white');
+const whiteOpacity20 = changeAlpha(white, 0.2);
```

Comment:
Use global app var `colorWhiteBackgroundOpacity` https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/d1de13093894012841bd1b4c51e486d97e093a4a/ride_to_work_by_bike_config.toml#L14 here instead of custom defined opacity number, please.

Reply:
Updated from global config.